docs/tutorials/policy_functions.ipynb

```diff
@@ -363,8 +304,7 @@
    "source": [
     "df = compute_taxes_and_transfers(\n",
     "    data=data,\n",
-    "    params=policy_params,\n",
-    "    functions=[policy_functions, arbeitsl_geld_2_m_bg, kindergeld_m],\n",
+    "    environment=environment.upsert_functions(arbeitsl_geld_2_m_bg, kindergeld_m),\n",
```

Comment:
I'd defer this part until the interface is finalized. Once parameters are also included in the DAG, this section might require updates.

Reply:
> I think the current default is that new functions are not vectorized. Imo it would make sense to change the default towards vectorization.

If users pass a `PolicyFunction`, they can set the `skip_vectorization` parameter appropriately. By default, it's `False`, so the function gets vectorized. If users pass another callable, a `PolicyFunction` is created, which always gets vectorized.

Reply:
No, because it would also replace (update) existing ones.

Although, thinking of it, Python's  `dict.update` has the same behavior, so maybe `update_policy_functions` is clear enough. I would indeed expand `functions`.

No reason to spend lots of time on it as things may change yet again once we include parameters in the DAG.